buckets/local/archive.go

```diff
@@ -111,12 +111,10 @@ func (b *Bucket) ArchiveStatus(ctx context.Context, watch bool) (<-chan ArchiveS
 					return
 				}
 				scancel()
-				final, err := isJobStatusFinal(r.GetStatus())
+				_, err = isJobStatusFinal(r.GetStatus())
 				if err != nil {
 					msgs <- ArchiveStatusMessage{Type: ArchiveError, Error: err}
 					cancel()
-				} else if final {
-					cancel()
```

Comment:
The problem here was that if they reach a _final_ status in a good way (success), the context of `ArchiveWatch` was indirectly canceled, so the logs were printed partially.